components/camel-quartz/src/main/docs/quartz-component.adoc

```diff
@@ -164,6 +164,22 @@ quartz://groupName/timerName?cron=0+0/5+12-18+?+*+MON-FRI&trigger.timeZone=Europ
 
 The timeZone value is the values accepted by `java.util.TimeZone`.
 
+== Specifying start date
+
+The Quartz Scheduler allows you to configure start date per trigger. You can provide the start date as TimeInMillis as follows:
+
+----
+quartz://groupName/timerName?cron=0+0/5+12-18+?+*+MON-FRI&trigger.startAt=1700452232554
+----
+
+== Specifying end date
+
+The Quartz Scheduler allows you to configure end date per trigger. You can provide the end date as TimeInMillis as follows:
+
+----
+quartz://groupName/timerName?cron=0+0/5+12-18+?+*+MON-FRI&trigger.endAt=1700452232554
+----
+
```

Comment:
It's probably important do add note here stating that, when using a system clock such as `System.currentTimeMillis()`, users may be susceptible to problems caused by time drifts (such as when the system clock gets automatically adjusted after an inquiry to an NTP server) and unpredictable behavior during daylight saving time changes. 

Ideally, we should provide a duration that can be computed from a monotonic clock in complement to this.

Reply:
Yes its also cumbersome to set a time as time millis - Its better its using date/timezone like you can do in simple language - also we should only do this in newer versions 4.3.0 and 3.22.0

Reply:
Is there any existing example for setting the time as date/timezone? I was inclined to use ISO-8601 but was thinking of something like "2001-07-04T12:08:56CET" for better readability, especially for DST based times.
I am presently using 3.14 due to some hard dependencies on Java 8. Hence I would urge you to also consider 3.14 as a candidate for shipping this feature.

Reply:
> Is there any existing example for setting the time as date/timezone? I was inclined to use ISO-8601 but was thinking of something like "2001-07-04T12:08:56CET" for better readability, especially for DST based times. I am presently using 3.14 due to some hard dependencies on Java 8. Hence I would urge you to also consider 3.14 as a candidate for shipping this feature.

I'd probably make this configurable.

Reply:
Can you share an example of the date format? I will look into it and submit the change

Reply:
I have replaced currentTimeMillis with date format "yyyy-MM-dd'T'HH:mm:ssz". Let me know if there is a better way to handle this case. Since cron does not support milliseconds I was inclined to the stated format.

Reply:
@orpiske @davsclaus can you please share your inputs/erviews?

Reply:
Thanks. I am fine with the approach. I would still like to see a note about time possible time drifts and inconsistent behavior during daylight changes (because it's caused by the nature of using time specifications instead of duration).

Reply:
Thanks. I have updated the adoc with a brief note.